force-app/main/default/classes/SlackServiceTest.cls

```diff
@@ -27,17 +27,17 @@ private with sharing class SlackServiceTest {
             Test.startTest();
             SlackService.postMessages(message);
             Test.stopTest();
-            System.assert(false, 'Exception expected');
+            Assert.isTrue(false, 'Exception expected');
         } catch (Exception e) {
             // THEN
-            System.assert(
+            Assert.isTrue(
                 e instanceof SlackService.SlackException,
                 'SlackException exception expected'
             );
             String expectedMessage =
                 'Failed to post messages to Slack: HTTP500 Failure - ' +
                 message;
-            System.assertEquals(
+            Assert.areEqual(
```

Comment:
Param order is incorrectL it's expected value first then actual.
You can also drop the custom message.